Header header

float32 min_range

float32 max_range 

float32 min_bearing 

float32 max_bearing 

float32 bearing
